js/templates/shared/toggle-button.js

```diff
@@ -0,0 +1,18 @@
+const bel = require('bel');
+
+module.exports = function (isActiveBoolean, klass, dataKey) {
+
+    // make `klass` and `dataKey` optional:
+    klass = klass || '';
+    dataKey = dataKey || '';
+
+    return bel`
+    <button class="toggle-button toggle-button--is-active-${isActiveBoolean} ${klass}"
+            data-key="${dataKey}"
+            type="button">
+        <div class="toggle-button__bg">
+        </div>
+        <div class="toggle-button__knob"></div>
+    </button>`;
```

Comment:
ah hm... i dont get that on my chrome version but i know how to fix it... no problem, thx for finding.